src/index.js

```diff
@@ -62,29 +62,31 @@ queuePromise.then(q => {
       console.log(data.run)
 
       const document = {
-        template: fs.readFileSync(templatePath).toString('utf-8'),
-        context: {
-          data
-        },
-        path,
-        options: {
-          type: "pdf",
-          format: "A4",
-          orientation: "landscape",
-          border: "0",
-          base: "file:///" + __dirname + '/assets/'
-        }
-      }
-
-      await PDF.create(document, {
-        phantomPath: p.join(__dirname, '../node_modules/phantomjs-prebuilt/bin/phantomjs')
-      })
+          template: fs.readFileSync(templatePath).toString('utf-8'),
+          context: {
+            data
+          },
+          options: {
+              format: 'A4',
+              landscape: true,
```

Comment:
- remove format and landscape option
- explicitly specify
    `height: '595px', width: '842px'`